packages/celotool/src/cmds/deploy/upgrade/verify-contracts.ts

```diff
@@ -0,0 +1,47 @@
+import * as yargs from 'yargs'
+import { InitialArgv } from '../../deploy/initial'
+import { switchToClusterFromEnv } from 'src/lib/cluster'
+import { getBlockscoutUrl } from 'src/lib/endpoints'
+import { execCmd } from 'src/lib/utils'
+
+export const command = 'verify-contracts'
+
+export const describe = 'verify the celo smart contracts in blockscout'
+
+export const builder = (argv: yargs.Argv) => {
+  return argv
+    .option('skipClusterSetup', {
+      type: 'boolean',
+      description: 'If you know that you can skip the cluster setup',
+      default: false,
+    })
+    .option('contract', {
+      type: 'string',
+      description: 'Contract name if only one contract want to be verified',
+      default: 'all',
+    })
+}
+
+interface VerifyContractsInitialArgv extends InitialArgv {
+  skipClusterSetup: boolean
+  contract: string
+}
+
+export const handler = async (argv: VerifyContractsInitialArgv) => {
+  await switchToClusterFromEnv()
+  // Check if blockscout is deployed and online?
+  const blockscoutUrl = getBlockscoutUrl(argv)
```

Comment:
not specified as arg

Reply:
The blockscout url can be get from the env, so we do not need to provide that as an arg. The url is propagated to the plugin as an argument so the plugin can use it.